### DISCUSSION TITLE
Flet talk on a flutter event

### DISCUSSION THREAD

**Discussion by TobiasHT5** [flet-dev/flet#1194] [Show and tell]
![IMG-20230318-WA0019.jpg](https://user-images.githubusercontent.com/35004147/226189310-293577fa-389b-4f8d-8fea-d507382a84b2.jpg)

I talked about how to build flutter apps with python and flet on one of our local meetups here in Kampala Uganda. It was really welcomed and we're starting a series of virtual and physical classes to take developers through flutter and flet

**Comment by FeodorFitsner** [comment]
That's amazing! Thank you guys for your support!
